app/src/common/shared/org/mozilla/vrbrowser/SessionStore.java

```diff
@@ -370,6 +380,19 @@ public String getUriFromSession(GeckoSession aSession) {
         return "";
     }
 
+    public String getTitleFromSession(GeckoSession aSession) {
+        Integer sessionId = getSessionId(aSession);
```

Comment:
perhaps could these five lines be replaced with a DRY method for `getStateFromSession`? I see this code repeated elsewhere.